src/components/icons/components/action/HistoryIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+import useIcon, { IconProps } from '../../../../hooks/useIcon';
+
+const History = (props: IconProps) => {
+    const iconFactory = useIcon(
+        <path
+            clip-rule="evenodd"
```

Comment:
JSX props should be camel cased (clipRule, fillRule etc.)

Reply:
Updated in the new commit